tools/gen_examples.py

```diff
@@ -412,9 +414,11 @@ def simple_example(draft):
       'ice_pwd': 'mqyWsAjvtKwTGnvhPztQ9mIf', 'dtls_dir': 'passive' }
   ]
   fp1 = '19:E2:1C:3B:4B:9F:81:E6:B8:5C:F4:A5:A8:D8:73:04:BB:05:2F:70:9F:04:A9:0E:05:E9:26:33:E8:70:88:A2'
+  tid1 = '91bbf309c0990a6bec11e38ba2933cee'
```

Comment:
Can these simply be monotonically increasing integers, for simplicity?

Reply:
They are required to have 120 bits of entropy, so they have to be long. but I could do H(integer) if you like.

Reply:
oh, ok. Do you know why? The dtls-sdp document doesn't provide any rationale for this.

Reply:
See https://github.com/cdh4u/draft-dtls-sdp/issues/36

Reply:
Luckily, we don't need to decide this for this document/PR because clearly long is OK. Do you want me to change them to be H(integer)?